demos/sample_azure_iot_ca_recovery/sample_azure_iot_ca_recovery.c

```diff
@@ -135,6 +143,36 @@ uint64_t ullGetUnixTime( void );
 
 static uint8_t ucPropertyBuffer[ 32 ];
 static uint8_t ucScratchBuffer[ 128 ];
+static uint8_t ucRootCABuffer[ 5000 ];
+static uint32_t ulRootCABufferWrittenLength;
+static uint8_t ucRootCATrustBundleVersion[ 16 ];
+static uint32_t ulRootCATrustBundleVersionLength;
+static uint8_t ucSignatureValidateScratchBuffer[ azureiotjwsSHA_CALCULATION_SCRATCH_SIZE ];
+
+static uint8_t ucAzureIoTRecoveryRootKeyN[] =
```

Comment:
make this a config so people can add their own. Also update the docs so that users know how to get this value easily.